deps/Makefile

```diff
@@ -740,6 +732,9 @@ PCRE_OBJ_TARGET = $(build_shlibdir)/libpcre2-8.$(SHLIB_EXT)
 
 # Force optimization for PCRE flags (Issue #11668)
 PCRE_CFLAGS = -O3
+ifneq ($(OS),WINNT)
+PCRE_LDFLAGS = "-Wl,-rpath,'$(build_libdir)'"
```

Comment:
Oh yeah, I feel like I saw that somewhere else too, the separate libraries weren't loading the interlibrary dependencies properly. Maybe libtool will get quieter now too, and the warnings were legit.